pipestat_example/pipeline/pipeline_interface_project.yaml

```diff
@@ -0,0 +1,6 @@
+pipeline_name: count_lines
```

Comment:
it's confusing to me that the sample pipeline and the project pipeline are the same.

you should actually write a simple project pipeline, not point to the sample pipeline.

a simple project pipeline could, for example, count the number of files in the project. or, count all the lines in the project.